source/includes/connection-snippets/scram-snippets/ruby-connection.rb

```diff
@@ -0,0 +1,5 @@
+# begin scram connection
+require 'mongo'
+
+client = Mongo::Client.new('mongodb+srv://kay:myRealPassword@cluster0.mongodb.net/?ssl=true&w=majority')
```

Comment:
The user/password/cluster placeholder `<>` used in the other drivers is missing here. Could you make this one consistent?

Reply:
Per https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst#seedlist-discovery `ssl=true` should be the default, and thus should not be needed.